content/millennium/r4/care-provision/service-request.md

```diff
@@ -0,0 +1,124 @@
+* [Body site](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.bodySite){:target="_blank"}
+* [Note](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.note){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:service_request, :r4) %>
+
+## Search
+
+Search for ServiceRequests that meet supplied query parameters:
+
+    GET /ServiceRequest?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name                     | Required?                       | Type          | Description
+--------------------------|---------------------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` or `subject` | [`token`]     | The logical resource id associated with the resource. Example: `12345`
+ `patient`                | This, or `_id` or `subject`     | [`reference`] | The specific patient to return ServiceRequests for. Example: `12345`
+ `subject`                | This, or `_id` or `patient`     | [`reference`] | The subject associated with the ServiceRequest. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/1316020` or `subject:Patient=1316020`
+ `_lastUpdated`           | N                               | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the service request. Must include a time, and must be prefixed by `ge` or `le`. Example: `ge2014-05-19T20:54:02.000Z`
+ [`_count`]               | N                               | [`number`]    | The maximum number of ServiceRequests to retrieve in a page. Example: `50`
+
+### Note
+
+  * If `_id` is provided, no other parameters may be provided.
+  * The `_lastUpdated` parameter may be provided:
+      * once with a prefix `ge` or `le` representing the earliest date or latest date. (e.g. `date=ge2015-01-01`, `date=le2016-01-01`)
+      * twice with the prefixes `ge`, `le` to indicate a specific range. (e.g. `date=ge2015-01-01&date=le2016-01-01`)
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/ServiceRequest?patient=1316020
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_SERVICE_REQUEST_BUNDLE) %>
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+List an individual ServiceRequest by its id:
+
+    GET /ServiceRequest/:id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Headers
+
+<%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/ServiceRequest/56770371
```

Comment:
The example data has ID: 41636441
Can we update 56770371 to 41636441 as well please.

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/389/commits/18a592885426eefd181f05aa020842939e654c6a